mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/plugins/DependencyInjection.kt

```diff
@@ -185,6 +189,20 @@ private fun services(appConfig: AppConfig) = module {
     single { TiltakstypeService(get()) }
 }
 
+private fun tasks(config: TaskConfig) = module {
+    single {
+        val synchronizeNorgEnheterTask = SynchronizeNorgEnheter(config.synchronizeNorgEnheter)
+
+        val db: Database by inject()
+
+        Scheduler
+            .create(db.getDatasource())
+            // .startTasks(synchronizeNorgEnheterTask.task)
```

Comment:
Har kommentert denne ut med vilje for å ikke skape støy i dev og prod når denne PRen merges, men vi ikke har noe fornuftig å kjøre i jobben enda.